src/services/jwk.service.ts

```diff
@@ -45,6 +46,21 @@ export class JWKService {
       logService.debug('Finished generating new keys!');
 
       JWKService.instance = new JWKService(keystore);
+
+      // add interval to rotate keys on instance creation
+      JWKService.instance.keyRotationInterval = setInterval(
+        () =>
+          JWKService.getInstance()
+            .then(jwkService => {
+              return jwkService.rotateKeys();
+            })
+            .then(() => logService.log('JWKService rotated keys!'))
+            .catch((err: unknown) => {
+              logService.log('An error occurred while the JWKService was rotating keys!');
+              logService.error(err);
+            }),
+        KEY_ROTATION_INTERVAL
+      );
```

Comment:
It occurs to me that I should check how evervault handles this - if enclaves are transparently horizontally scaled and load-balanced, this approach will cause problems.

Reply:
@K-Mistele An additional question: Could you please clarify where the keys should be stored?

Reply:
Yes, after my meeting with evervault I should have a better idea of how to approach this )